### PR TITLE
Create a risk analysis plan

### LINKED ISSUE #3: Create a Risk analysis plan.
Create a Risk analysis for the project

### PR DESCRIPTION
**Pull request by MBRzealand**
Co-authored-by: Madictian <70747015+Madictian@users.noreply.github.com>
Closes #3

### CODE REVIEW

**Review by Hjordrup — CHANGES_REQUESTED** [review]
There may need to look at the first priority cause not going to the exam may be a very big problem,

File: 09 Environment/RiskAnalysis.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    1-2  : very low  
+    3-4  : low  
+    5-6  : medium  
+    7-8  : high  
+    9-10 : very high  
+
+
+|  Risk | Probability | Consequence | Priority | Revised Probability | Revised Consequence |
+| ------------------------------------ | ---------------------------------------------------- | --------- | --------- | ------------------ |--------- |
+| Unable to deliver complete app due to priorities | 3 | Unable to live up to client expectations | 5 | 2 | Unable to deliver report |
```
Comment (Hjordrup) [inline 09 Environment/RiskAnalysis.md:14]: You may need to look at the first priority, cause not going to the exam may be a very big problem.

**Review by Hjordrup — CHANGES_REQUESTED** [review]
The first priority is very low in my opinion

File: 09 Environment/RiskAnalysis.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+**Counter Strategy:**  
+Allocating less time to write the report would allow us more time for app development, which in turn would make us able to deliver a more functional app.
```
Comment (Bombohog) [inline 09 Environment/RiskAnalysis.md:17]: Revise the counter, so the report isnt hurt as bad

**Review by Bombohog — CHANGES_REQUESTED** [review]

**Review by Bombohog — APPROVED** [review]
I can approve this, this is better than not being able to deliver the report.

**Review by Hjordrup — APPROVED** [review]

**Pull request merged**